### PR TITLE
Added Arrows+, Attachment and Download Icons

### PR DESCRIPTION
**Pull request by binayshaw7777**
This PR Adds 6 Icons -> [Source](https://chakra-ui.com/docs/components/icon)

Icons added:
- [x] ArrowBackIcon
- [x] ArrowDownIcon
- [x] ArrowForwardIcon
- [x] ArrowUpIcon
- [x] AttachmentIcon
- [x] DownloadIcon

### CODE REVIEW

File: frontend/silk-widgets/src/jsMain/kotlin/com/varabyte/kobweb/silk/components/icons/Icons.kt [hunk 1]
```diff
@@ -47,6 +48,124 @@ private fun createIcon(
     }, content)
 }
 
+//Attachment,Close,Copy,
```
Comment (bitspittle) [inline frontend/silk-widgets/src/jsMain/kotlin/com/varabyte/kobweb/silk/components/icons/Icons.kt]: Nit: Please delete these comments, or possibly, put them in a TODO

A  TODO would look something like this:

```
// TODO(#306): Add remaining icons
//  Attachment
//  Close
//  Copy
//  ...
```
^ Notice that the lines following the TODO start with a double space! This tells the IDE to continue the TODO comment across multiple lines.

That said, I'm OK if you just delete the comment entirely, and move the list of icons to the github issue itself in a comment. Up to you!

But

Comment (bitspittle) [inline frontend/silk-widgets/src/jsMain/kotlin/com/varabyte/kobweb/silk/components/icons/Icons.kt]: Also, note that "Close" and "Hamburger" are now done.

File: playground/site/src/jsMain/kotlin/playground/pages/Widgets.kt [hunk 2]
```diff
@@ -202,13 +208,19 @@ fun WidgetsPage() {
 
             WidgetSection("Icons") {
                 val icons = mapOf<String, @Composable () -> Unit>(
+                    "ArrowBackIcon" to { ArrowBackIcon() },
```
Comment (bitspittle) [inline playground/site/src/jsMain/kotlin/playground/pages/Widgets.kt:211]: Nit: Change names
* "Arrow Back"
* "Arrow Forward"
* ...
* "Download"

**Review by bitspittle — CHANGES_REQUESTED** [review]
I love how the type-safe `d { ... }` block looks.

File: frontend/silk-widgets/src/jsMain/kotlin/com/varabyte/kobweb/silk/components/icons/Icons.kt [hunk 3]
```diff
@@ -48,10 +48,6 @@ private fun createIcon(
     }, content)
 }
 
-//Attachment,Close,Copy,
```
Comment (bitspittle) [inline frontend/silk-widgets/src/jsMain/kotlin/com/varabyte/kobweb/silk/components/icons/Icons.kt]: Oh, hahah, you deleted the comments here. That's great.

So just change the names in the Widgets.kt file and I'll squash and merge.

**Review by bitspittle — APPROVED** [review]

File: frontend/silk-widgets/src/jsMain/kotlin/com/varabyte/kobweb/silk/components/icons/Icons.kt (hunk 1 above)
Comment (binayshaw7777) [reply frontend/silk-widgets/src/jsMain/kotlin/com/varabyte/kobweb/silk/components/icons/Icons.kt]: I actually removed the comments in the last commit, isn't that updated here? 🤔

File: playground/site/src/jsMain/kotlin/playground/pages/Widgets.kt (hunk 2 above)
Comment (binayshaw7777) [reply playground/site/src/jsMain/kotlin/playground/pages/Widgets.kt:211]: Sure

File: frontend/silk-widgets/src/jsMain/kotlin/com/varabyte/kobweb/silk/components/icons/Icons.kt (hunk 1 above)
Comment (binayshaw7777) [reply frontend/silk-widgets/src/jsMain/kotlin/com/varabyte/kobweb/silk/components/icons/Icons.kt]: Noted

Comment (binayshaw7777) [reply frontend/silk-widgets/src/jsMain/kotlin/com/varabyte/kobweb/silk/components/icons/Icons.kt]: I too checked the last comment, 😅 please ignore the above comment, marking resolved

**Pull request merged**